### ISSUE TITLE
[Bug Report][2.4.9] Hidden Items in Virtical stepper are discoverable by keyboard and screenreader 

### ISSUE THREAD

**Issue by flpehr0** [vuetifyjs/vuetify#14028]
### Environment
**Vuetify Version:** 2.4.9
**Vue Version:** 2.6.12
**Browsers:** Chrome 92.0.4515.107, Google Chrome, Mozilla Firefox
**OS:** Windows 10

### Steps to reproduce
Begin with a screen reader turned on (I used NVDA). Attempt to navigate the vertical stepper with the keyboard using both the arrow keys and the tab key. Listen for the items that the keyboard is accessing with the screen reader. 

### Expected Behavior
The screen reader and keyboard should not be able to access or read hidden items in the v-stepper-content when they are not on that step. 

### Actual Behavior
you will be able to navigate and execute actions in the hidden v-stepper-content. In our app I was able to fill out entire portions of a form that were not yet visible. 

### Reproduction Link
<a href="https://codepen.io/flpehr0/pen/ExmeojB?editors=101" target="_blank">https://codepen.io/flpehr0/pen/ExmeojB?editors=101</a>

### Other comments
- This does not seem to be a problem for horizontal steppers that I could tell when testing. 
- As a general rule of thumb things that are hidden from some users should be hidden from all users. 
- I attempted to use aria-hidden as well as v-if on the v-stepper-content to make the hidden items inaccessible to the keyboard and the screen reader and it did not work.

**Comment by lioneaglesolutions** [comment]
### Stale Issue

This issue is being closed due to inactivity.

If you have any additional questions, please reach out to us in our [[Discord community](https://community.vuetifyjs.com/)](https://community.vuetifyjs.com/).
